integration-tests/rest-client-reactive/src/main/java/io/quarkus/it/rest/client/main/FaultToleranceOnInterfaceClient.java

```diff
@@ -12,7 +12,7 @@
 import org.eclipse.microprofile.rest.client.inject.RegisterRestClient;
 
 @Path("/unprocessable")
-@RegisterRestClient(configKey = "w-fault-tolerance")
+@RegisterRestClient(configKey = "w-fault-tolerance-int")
```

Comment:
This change was made in order to make the client have a config key that is not defined anywhere